src/lib/template_style/_theme.scss

```diff
@@ -44,6 +44,16 @@
   }
 }
 
+.SRC-secondary-background-color-hover:hover {
```

Comment:
copy-paste of the above class, which is primary color hover. This will be used in portals.